process-engine/bpmn-studio
# 🐛 Fix Authorization for Web Version

## Changes

1. Fix Login for Web Version
2. Fix Logout for Web Version

## Issues

Closes #1917

PR: #1920

## How to test the changes

- Open the web version of the BPMN Studio
- Start a ProcessEngine
- Start an IdentityServer
- Click on the login button
- **Notice that the login page gets displayed in a popup.**
- Login
- Open the developer console
- Go to the `Network` tab
- Select any entry
- **Notice that the dummy token is NOT used anymore.**
-  Click on the logout button
- **Notice that the logout page gets displayed in a popup.**
- Logout
- Open the developer console
- Go to the `Network` tab
- Select any entry
- **Notice that the dummy token is being used.**
